src/components/Article/Article.jsx

```diff
@@ -0,0 +1,42 @@
+import articleData from "../../translations/enArticleData.json";
+import { ArticleCard } from "./ArticleCard";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+
+const ArticleWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 10px;
+  max-width: 100%;
+  padding: 38px 15px;
+  @media (min-width: 768px) {
+    
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 1fr 1fr 1fr;
+   width: 1283px;
+   margin: 0 auto;
+  }
+`;
+
+let articleArray = articleData.articles;
+console.log(articleArray);
+
+export const Article = () => {
+  const { t, ready } = useTranslation();
+
+  const articles = t("articles", { returnObjects: true });
+
+  if (!ready) return "loading translations...";
```

Comment:
I haven't started using translations just yet. But I am impressed by the code. Really clear to keep the loading-time communicated to the user!